### ISSUE TITLE
Midpoint Review - Accessibility Feedback - [Employee Experience, 686c, Fully Digital Form Viewer MVP]

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#125252]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Thoughts/questions

- [ ] **Consider:** Consider how the summary page information is going to be constructed to ensure users of assistive technology are able to understand the experience. 
    - Is it important that the list be one ordered list (`<ol>`) to provide the numbering, or is the numbering acting as more of a reference label for internal end users? 
        - If the former, as the complexity of formatting grows, the potential for assistive technology to inaccurately parse the experience increases. A simple demo of an interrupted ordered list appears to have some minor issues that likely wouldn't equate to logged defects, but as complexity grows via web components, additional nesting, etc. these issues could grow. 
            - Demo: https://codepen.io/jasonday/pen/zxqvjmq
                - JAWS/Chrome announces the number of list items correctly.
                - Voiceover/Safari includes other elements, such as headings, as being included in the list (e.g. "heading level 3, Category 1, 1 of 16").
                - NVDA/Firefox includes other elements, such as headings, as being included in the list count (e.g. "heading level 3, Category 1, 1 of 16"). Additionally, navigating by next list item ("i" key with NVDA on) fails on the list items wrapped in a `<div>`) - "No next list item"
            - Talkback/Android announces list as expected
        - If the latter, then separate unordered lists could be effective and lessen the chances for less than ideal screen reader experiences
- [ ] **Consider:** With TAB/columnar/menu style spacing, there is the potential for some users to struggle to track across the empty white space, especially with items that have more horizontal white space and/or less vertical white space. Users with visual impairments such as an astigmatism, or users that zoom in at 400% (with Zoom text), may not be able to track effectively. In areas where the data is commonly related, such as an address, this may not be an issue as the user may be able to build context from familiarity. But, when the data is not familiar, or has less context to surrounding lines, this may get more difficult. 
    - Adding some vertical whitespace, and styling, such as an underline would be one potential solution to this issue. I realize that with 100+ items, this could get unwieldy, and may lead towards needing to collapse some content to make it more usable/less overwhelming for users.
    - Further, for future state (as a component, solution), it may be worth looking into use of grid, if there's the potential for multiple data points (table style layout, but with more control, especially for small screens) @humancompanion-usds & @artsymartha68 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- 

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app
